lib/collections/collectionFS.js

```diff
@@ -13,80 +22,77 @@ FS.HTTP.setHeadersForGet([
   ["Cache-Control", "public, max-age=31536000"]
 ]);
 
+/**
+ * @name imgTransforms
+ * @constant {Array}
+ * @property {string} name - transform name that will be used as GridFS name
+ * @property {object|undefined} transform - object with image transform settings
+ * @property {number} size - transform size, only one number needed for both width & height
+ * @property {string} helper - transform helper function call,
+ * for example the `large` & `medium` image transforms want to preserve
+ * the image's aspect ratio and resize based on the larger width or height
+ * so we use the `max` Sharp helper function.
+ * Check out the {@link http://sharp.pixelplumbing.com/en/stable/|Sharp Docs} for more helper functions.
+ * {@link http://sharp.pixelplumbing.com/en/stable/api-resize/#max|Sharp max()}
+ * {@link http://sharp.pixelplumbing.com/en/stable/api-output/#png|Sharp png()}
+ * @summary Defines all image transforms
+ * Image files are resized to 4 different sizes:
+ * 1. `large` - 1000px by 1000px - preserves aspect ratio
+ * 2. `medium` - 600px by 600px - preserves aspect ratio
+ * 3. `small` - 235px by 235px - crops to square - creates png version
+ * 4. `thumbnail` - 100px by 100px - crops to square - creates png version
+ */
+const imgTransforms = [
+  { name: "image" }, // creates the first `image` GridFS store
+  { name: "large", transform: { size: 1000, helper: "max" } }, // max helper
+  { name: "medium", transform: { size: 600, helper: "max" } }, // max helper
+  { name: "small", transform: { size: 235, helper: "png" } }, // png helper
+  { name: "thumbnail", transform: { size: 100, helper: "png" } } // png helper
```

Comment:
For "small" and "thumbnail", the existing transforms have "^" appended, and gm docs say:

> Append a ^ to the geometry so that the image aspect ratio is maintained when the image is resized, but the resulting width or height are treated as minimum values rather than maximum values.

So I think helper should be `"min"` on those, and then "png". 

The "extent" should not be necessary for those I would think, since it will fill the square. I'm not sure why they have `.gravity("Center")` on them, but @aaronjudd might know whether we can safely omit that.